### DISCUSSION TITLE
Option to hide system tray icon

### DISCUSSION THREAD

**Discussion by CSingh97** [mullvad/mullvadvpn-app#9081] [Feature requests & Ideas]
When you start up Mullvad, an icon will show in your system tray. It'd be great if there was an option to hide this icon since Gnome has extensions to indicate you're connected through the vpn, and in Windows there's an icon to indicate you're connected through a network.

Operating system: Fedora 36 Workstation

App version: 2022.5-beta1

**Comment by SarcevicAntonio** [comment]
This would be greatly appreciated, especially since the recommended workaround using "AppIndicator and KStatusNotifierItem Support" sometimes results in "three dots" placeholder icon, making the tray icon basically useless :(
